docs/contributing-to-analyses/pr-review-and-merge/respond-to-review.md

```diff
@@ -0,0 +1,171 @@
+
+</div>
+
+
+## Types of reviewer comments
+
+When reviewing your pull request, there are several types of comments your reviewer may leave for you to respond to.
+
+!!! tip "Reading review comments"
+    Unresolved file-level and in-line review comments will appear both in the `Conversation` and in the `Files Changed` tabs.
+
+    - In the `Conversation` tab, comments are listed in chronological order based on when the reviewer wrote the comment, which might not match the order of the code itself.
+    - In the `Files Changed` tab, comments appear right at the file or line of code they are about.
+
+
+### Comments with overall feedback
+
+In most circumstances, your reviewer will leave an overall high-level comment explaining their overall impressions from reviewing the code and synthesizing the types of changes they are requesting you make.
+
+### File-level comments
+
+Your reviewer can leave file-specific comments that provide high-level feedback about the overall contents of a given file.
+
+For example, this file-level review comment requests that the code author comment and document a script:
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-02.png){width="600"}
+</figure>
+
+
+### In-line comments
+
+Your reviewer can also leave in-line comments with feedback about specific lines of code:
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-03.png){width="600"}
+</figure>
+
+You can directly reply to these review comments on GitHub.
+For example, you might ask your reviewer to clarify a particular comment, or ask for more guidance about how to implement the review.
+Or, you might just want to leave your reviewer a helpful note of some kind!
+
+The way to responding to an existing comment differs depending on which PR tab you are leaving the comment from:
+
+- If you are in the **`Files changed`** tab, use the button `Add single comment` to reply (_do not use `Start a review`_):
+    <figure markdown="span">
+        ![Reply to a review comment from the `Files changed` tab.](../../img/respond-to-review-04.png){width="600"}
+    </figure>
+
+- If you are in the **`Conversation`** tab, use the button `Comment`.
+    <figure markdown="span">
+        ![Reply to a review comment from the `Conversation` tab.](../../img/respond-to-review-05.png){width="600"}
+    </figure>
+
+
+### Suggestion comments
+
+Reviewer comments can also include _suggestions_ for code changes, that you can incorporate into your PR directly on GitHub.
+
+For example, below are two suggestions left as comments on an R script:
+
+- In the first comment, the reviewer suggested that the code author add a new line to load the `readr` library.
+- In the second comment, the reviewer suggested that the code author remove a line that loads the `dplyr` library.
+
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-06.png){width="600"}
+</figure>
+
+
```

Comment:
```suggestion
## Responding to review comments 

### Responding to suggestion comments 
```
Everything before this is really examples, and here you are talking about how to actually incorporate suggestions and respond. So I think you need to move up the header.